internal/common/host_utils.go

```
@@ -17,3 +17,11 @@ func GetCurrentHostName(host *models.Host) (string, error) {
 	}
 	return inventory.Hostname, nil
 }
+
+func GetHostnameForMsg(host *models.Host) string {
+	hostName, err := GetCurrentHostName(host)
```

We want to ignore this error? Maybe we should log it at least?

---

It means that the host doesn't have inventory yet, so just fall back to using the ID

---

Please add a comment

---

Done